src/temporal/t.rast.boxplot/t.rast.boxplot.py

```diff
@@ -0,0 +1,620 @@
+# % description: Draw boxplot(s) with outliers
+# % guisection: Statistics
+# %end
+
+# %flag
+# % key: n
+# % label: Draw notch
+# % description: Draw boxplot(s) with notch
+# % guisection: Statistics
+# %end
+
+# %option
+# % key: range
+# % type: double
+# % label: Range (value > 0)
+# % description: This determines how far the plot whiskers extend out from the box. If range is positive, the whiskers extend to the most extreme data point which is no more than range times the interquartile range from the box. A value of zero causes the whiskers to extend to the data extremes.
+# % required: no
+# % answer: 1.5
+# % guisection: Statistics
+# %end
+
+# %option
+# % key: plot_dimensions
+# % type: string
+# % label: Plot dimensions (width,height)
+# % description: Dimensions (width,height) of the figure in inches
+# % required: no
+# % guisection: Plot format
+# %end
+
+# %flag
+# % key: h
+# % label: Horizontal boxplot(s)
+# % description: Draw the boxplot horizontal
+# % guisection: Plot format
+# %end
+
+# %option
+# % type: double
+# % options: 0-365
+# % key: rotate_labels
+# % label: Rotate labels
+# % description: Rotate labels (degrees)
+# % guisection: Plot format
+# %end
+
+# %flag
+# % key: g
+# % label: Add grid lines
+# % description: Add grid lines
+# % guisection: Plot format
+# %end
+
+# %option
+# % key: fontsize
+# % type: integer
+# % label: Font size
+# % description: Default font size
+# % guisection: Plot format
+# % required: no
+# %end
+
+# %option
+# % key: date_format
+# % type: string
+# % label: Date format
+# % description: Set date format (see https://strftime.org/ for options)
+# % guisection: Plot format
+# % required: no
+# %end
+
+# %option
+# % key: boxplot_width
+# % type: double
+# % label: Boxplot width
+# % description: The width of the boxplots
+# % required: no
+# % guisection: Boxplot format
+# %end
+
+# %options G_OPT_CN
+# % key: bxcolor
+# % type: string
+# % label: Boxplot color
+# % description: Color of the boxplots. See manual page for color notation options.
+# % required: no
+# % guisection: Boxplot format
+# %end
+
+# %option
+# % key: flier_marker
+# % type: string
+# % label: Flier marker
+# % description: Set flier marker (see https://matplotlib.org/stable/api/markers_api.html for options)
+# % required: no
+# % guisection: Boxplot format
+# %end
+
+# %option
+# % key: flier_size
+# % type: string
+# % label: Flier size
+# % description: Set the flier size
+# % required: no
+# % guisection: Boxplot format
+# %end
+
+# %option G_OPT_C
+# % key: flier_color
+# % label: Flier color
+# % description: Set the flier color
+# % required: no
+# % guisection: Boxplot format
+# %end
+
+
+import sys
+import atexit
+import uuid
+from subprocess import PIPE
+from dateutil import parser
+import matplotlib.dates as mdates
+import grass.script as gs
+from grass.pygrass.modules import Module
+
+
+clean_layers = []
+
+
+def create_unique_name(name):
+    """Generate a temporary name which contains prefix
```

Comment:
Func doc string:

```python
    """Generate a temporary name which contains prefix
    Store the name in the global list.

    :param str name: name param description

    :return str: return value description
    """
```

Reply:
Added extra information, using format that was recommended on https://www.programiz.com/python-programming/docstrings